### DISCUSSION TITLE
Renovate build fails with get config.js on Node 16 (Windows/node 16/Custom config.js)

### DISCUSSION THREAD

**Discussion by DutchBlackOp** [renovatebot/renovate#21702] [Suggest an Idea]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

31.82.1

### Please select which platform you are using if self-hosting.

Azure DevOps Server

### If you're self-hosting Renovate, tell us what version of the platform you run.

Version Azure DevOps Server 2020 Update 1.1

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

Renovate starting failing after updating from NodeJs 14.18.2 -> 16.13.0

The build started failing with the following error:
```
  DEBUG: Checking for config file in C:\build\Renovate/config.js
  DEBUG: No config file found on disk - skipping
```

After adding environment variable to specify the config.js path:
```
  DEBUG: Checking for config file in C:\build\Renovate\config.js
  FATAL: No custom config file found on disk
```

Trying to reproduce the code in here: https://github.com/renovatebot/renovate/blob/main/lib/workers/global/config/parse/file.ts#L67
I ran Node manually and tried to import: "C:\build\Renovate\config.js" manually.
- This fails because it's expecting a file url
I ran Node manually and tried to import: "file://C:\build\Renovate\config.js" manually.
- This works

After this test I tried to specify the environment variable to specify the config.js path to:
- "file://C:\build\Renovate\config.js"
The actual output in the build is:
- "C:\build\Renovate\file://C:\build\Renovate\config.js"

The library parsing the path does not allow me to set a fixed path with the "file://" prefix.

Can you reproduce this scenario and if so maybe allow for fixed paths that start with the "file://" prefix.

Let me know if you need any additional information.

Kind regards,
Stephan

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
I think we'll need a Windows user to contribute a fix for this
